files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml

```diff
@@ -63,3 +63,31 @@ destinations:
     scheduling:
       require:
         - pulsar
+
+  condor_singularity_with_conda:
+    runner: "condor"
+    cores: 64
+    mem: 1000
+    scheduling:
+      require:
+        - condor_singularity_with_conda
+    env:
+      GALAXY_MEMORY_MB: "{MEMORY_MB}"
+      GALAXY_SLOTS: "{PARALLELISATION}"
+      LC_ALL: C
+    params:
+      priority: -{PRIORITY}
+      request_cpus: "{PARALLELISATION}"
+      request_memory: "{MEMORY}"
+      requirements: 'GalaxyGroup == "compute"'
+      tmp_dir: "True"
+      metadata_strategy: "extended"
+      require_container: true
+      singularity_enabled: true
+      # can and should be made stricter at some point
+      singularity_volumes: "$_CONDOR_SCRATCH_DIR:rw,$job_directory:rw,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/data/db/:ro,/data/dnb01/galaxy_db/:ro,/data/dnb02/galaxy_db/:ro,/data/dnb03/galaxy_db/:ro,/data/dnb05/galaxy_db/:ro,/data/dnb06/galaxy_db/:rw,/data/dnb07/galaxy_db/:rw,/data/dp01/galaxy_db/:rw,/data/0/galaxy_db/:ro,/data/1/galaxy_db/:ro,/data/2/galaxy_db/:ro,/data/3/galaxy_db/:ro,/data/4/galaxy_db/:ro,/data/5/galaxy_import/galaxy_user_data/:ro,/data/6/galaxy_db/:ro,/data/7/galaxy_db/:ro,/usr/local/tools/:ro"
+      container_override:
```

Comment:
I'm not sure TPV will take this ... maybe but there is no other way than testing it :)

Reply:
According to this page, all entities should share these [https://total-perspective-vortex.readthedocs.io/en/latest/topics/concepts.html](https://total-perspective-vortex.readthedocs.io/en/latest/topics/concepts.html)